### PR TITLE
Update ex7_31.h

### PR DESCRIPTION
**Pull request by weixiaoqiang**
a question

### CODE REVIEW

**Comment by weixiaoqiang** [comment]
一个小问题

File: ch07/ex7_31.h [hunk 1]
```diff
@@ -6,8 +6,11 @@
 //  Copyright (c) 2014 pezy. All rights reserved.
 //
 
-#ifndef CP5_ex7_31_h
+#ifndef CP5_ex7_31_h    
 #define CP5_ex7_31_h
+/*i use #ifndef 7_31_H and there is an error:macro names must be identifiers
```
Comment (pezy) [inline ch07/ex7_31.h]: The standard (C11 6.4.2 Identifiers) mandates that identifiers **must not start with a digit**.

Comment (weixiaoqiang) [inline ch07/ex7_31.h]: thanks

**Comment by Mooophy** [comment]
@pezy  Agree.
@weixiaoqiang If no more discussion, can you please close this pull request for keep this repo clean?Thx

**Pull request closed without merging**